### ISSUE TITLE
d3.geoRotation

### ISSUE THREAD

**Issue by chrisuehlinger** [d3/d3-geo#13]
https://github.com/d3/d3/blob/v3.5.17/src/geo/rotation.js

**Comment by chrisuehlinger** [comment]
Just a heads up, we're going to run into a circular dependency between `d3.geoRotation`, `equirectangular` and `d3.geoProjection`. If we're planning on breaking `d3.geoProjection` into a separate node module, that would also mean that people importing these modules could end up with a circular dependency (which could be problematic depending on how they bundle/load).

It's looking like the most straightforward way to resolve all of this is to keep `d3.geoProjection` in `d3-geo` (along with the "batteries included" projections like `spherical` and `equirectangular`, which are required in components like `d3.geoPath` and `d3.geoCircle`) and reserve `d3-geo-projection` for the more exotic projections (basically, the way things are now).

If you feel strongly that we should separate `d3.geoProjection` into `d3-geo-projection`, I'm cool with making that the focus of my next large effort. However, I do wonder whether there's much of a use case for `d3-geo` without any projections (you can't really render a map without them).

**Comment by mbostock** [comment]
We’ll break the circular dependency.

d3.geoRotation only depends on equirectangular out of convenience to define the inverse rotation. Instead of this:

```js
d3_geo_identityRotation.invert = d3_geo_equirectangular;
```

It should be sufficient to say this:

```js
d3_geo_identityRotation.invert = d3_geo_identityRotation;
```

**Comment by mbostock** [comment]
Fixed in #18 by @chrisuehlinger.
